src/ui/server/constants/regex.ts

```diff
@@ -0,0 +1,5 @@
+export const REGEX = {
+  ALPHA_CHARACTERS: /^[a-zA-Z ]*$/,
```

Comment:
This is actually alpha characters and an empty space, same with the below regexes

Shall we add some comments to each regex? I think quite helpful, e.g

```js
/**
  * ALPHA_CHARACTERS:
  * Regex that allows only:
  * - a-z or A-Z characters.
  * - an empty space.
 */
  ALPHA_CHARACTERS: /^[a-zA-Z ]*$/,
```